deployments/cardano-prod.nix

```diff
@@ -6,13 +6,24 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["env:production"];
 
     # DEVOPS-64: disable log bursting
     services.journald.rateLimitBurst = 0;
 
     # Initial block is big enough to hold 3 months of transactions
     deployment.ec2.ebsInitialRootDiskSize = mkForce 700;
+
+    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.processConfig = ''
+      init_config:
+
+      instances:
+      - name: cardano-node
+        search_string: ['cardano-node']
+        exact_match: False
+        thresholds:
+          critical: [1, 2]
+    '';
```

Comment:
Neither `464c79ea9f929d1237dbc2df878eedad91767a72`, `ebbababd8f9cb49d039f11d58e4e49d8e02d7533`, nor `763e21e982370f67c126f92a1113ea949db3b6e0` do not appear to have `dd-agent.processConfig` either.. 

..and now, I see that `processConfig` is added by you, very recently, in `0bc033117f190bafc139995b0c57f618bb5ddf27`.

But at least I see now what `processConfig` is supposed to do : -)